walk-in-interview/src/main/java/com/google/account/business/data/BusinessDatabase.java

```diff
@@ -0,0 +1,21 @@
+package com.google.account.business.data;
+
+import com.google.cloud.firestore.WriteResult;
+import com.google.utils.FireStoreUtils;
+
+import java.io.IOException;
+import java.util.concurrent.Future;
+
+/** Helps persist and retrieve business accounts in cloud firestore. */
+public final class BusinessDatabase {
+    private static final String BUSINESS_ACCOUNT_COLLECTION = "BusinessAccounts";
+
+    // Creates a new business object and stores into the cloud firestore.
+    public Future<WriteResult> createBusinessAccount(String uid, Business business) throws IOException {
```

Comment:
Is this create or update?
If create, can we create two business with the same id? Would the future fail?

Reply:
if with the same uid, the later one will overlap the first one